extensions/ql-vscode/src/language-context-store.ts

```diff
@@ -46,4 +46,11 @@ export class LanguageContextStore extends DisposableObject {
   public shouldInclude(language: QueryLanguage | undefined): boolean {
     return this.languageFilter === "All" || this.languageFilter === language;
   }
+
+  public selectedLanguage(language: QueryLanguage | undefined): boolean {
```

Comment:
Could this be more descriptive of what it's actually checking? Something like `isSelectedLanguage`?